benchmark/README.md

```diff
@@ -0,0 +1,48 @@
+# Sim Benchmark
+
+A simple benchmark for pygazebo and pybullet simulator.
+
```

Comment:
Please note that pygazebo result is for bullet engine. We should also include benchmark of ODE for bullet.

Reply:
Done.